src/main/java/core/TrelloProperties.java

```diff
@@ -0,0 +1,25 @@
+package core;
+
+import java.io.IOError;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class TrelloProperties {
+    protected static Properties testdataProperties = new Properties();
```

Comment:
Should be public static to be seen and accessed from everywhere
the variable name - testDataProperties

Reply:
Fixed.